cluster/manifests/kubenurse/daemonset.yaml

```diff
@@ -56,11 +56,11 @@ spec:
           image: "container-registry.zalando.net/teapot/kubenurse:v1.7.0-histogram-main-5"
           resources:
             requests:
-              cpu: "40m"
-              memory: "100Mi"
+              cpu: "{{.Cluster.ConfigItems.network_monitoring_cpu}}"
```

Comment:
I think the name is `network_monitoring_daemonset_cpu` same for memory?

Reply:
fixed